core/src/main/java/tel/schich/javacan/platform/Platform.java

```diff
@@ -35,6 +37,8 @@
  * Helper class to detect and handle various platforms. Currently only Linux is handled.
  */
 public class Platform {
+    private static final Logger LOG = LoggerFactory.getLogger(Platform.class);
```

Comment:
super nitpicky, but: I'd prefer LOGGER as the name, it better fits the API. If you don't have time for this change, I'd be fine to merge this and rename it myself.

Reply:
No problem at all ;) I usually use the way it is in the project already, but in my private projects I tend to use the shorter version. ;)